frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -0,0 +1,201 @@
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findCancelButton().click();
+    createProjectModal.shouldBeOpen(false);
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findSubmitButton().should('be.disabled');
+  });
+
+  it('should create project', () => {
+    initIntercepts();
+    initCreateProjectIntercepts();
+
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.findNameInput().type('My Test Project');
+    createProjectModal.findDescriptionInput().type('Test project description.');
+    createProjectModal.findSubmitButton().should('be.enabled');
+    createProjectModal.findResourceNameInput().should('have.value', 'my-test-project').clear();
+    createProjectModal.findResourceNameInput().should('have.attr', 'aria-invalid', 'true');
+    createProjectModal.findSubmitButton().should('be.disabled');
+    createProjectModal.findResourceNameInput().type('test-project');
+
+    cy.intercept(
+      {
+        method: 'GET',
+        pathname: '/api/k8s/apis/project.openshift.io/v1/projects',
+      },
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    ).as('createdProjects');
+
+    createProjectModal.findSubmitButton().click();
+
+    cy.wait('@createProjectRequest').then((interception) => {
+      expect(interception.request.body).to.eql({
+        apiVersion: 'project.openshift.io/v1',
+        kind: 'ProjectRequest',
+        metadata: {
+          name: 'test-project',
+        },
+        description: 'Test project description.',
+        displayName: 'My Test Project',
+      });
+    });
+    cy.wait('@createdProjects');
+    cy.url().should('include', '/projects/test-project');
+  });
+
+  it('should list the new project', () => {
+    initIntercepts();
+    cy.intercept(
+      {
+        method: 'GET',
+        pathname: '/api/k8s/apis/project.openshift.io/v1/projects',
+      },
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    );
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.findProjectScopeDropdown().click();
+    projectListPage.findDatascienceProjectOption('Data science projects').should('be.visible');
+    projectListPage
+      .findDatascienceProjectOption(
+        'Only projects created in the Data Science Dashboard are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.findAllprojectOption('All available projects').should('be.visible');
+    projectListPage
+      .findAllprojectOption(
+        'All available projects created in the Data Science Dashboard and OpenShift are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.findProjectScopeDropdown().click();
+    projectListPage.findCreateProjectButton().should('exist');
+    projectListPage.findJupyterLauncher().should('exist');
```

Comment:
@christianvogt ,I believe the testing would cover the existence of UI Elements as well. All texts which are validating in the test cases are static text. A form shall be meaningless if the supporting field names are not present. Recently there was a blocker issue in a release https://issues.redhat.com/browse/RHOAIENG-2041 for not displaying text in UI and test case is  updated accordingly here https://github.com/opendatahub-io/odh-dashboard/blob/6913d3fd2fca00a00d1b254154e9cf924c35dbef/frontend/src/__tests__/cypress/cypress/e2e/applications/explore.cy.ts#L21. If we have test cases to cover all the elements, proactively we could identifying those issues in earlier stages.

I would prefer to cover all the text/field name validations to be covered along with functinality. The test cases should fail if any changes happens by mistake.
We can exclude these validations only if we are planning for seperate UX/UI test cases.

Reply:
The point I'm trying to make is that the presence of this action should be tested via a workflow that makes use of the action rather than arbitrarily checking the presence of a button without taking further action.

This test is focused on `should list the new project` and its reaching to assert other parts of the page which should be covered by other tests.